components/modal/src/modal/modal.js

```diff
@@ -7,67 +7,60 @@ import PropTypes from 'prop-types'
 import React from 'react'
 import { resolve } from 'styled-jsx/css'
 
-const scrollBoxCard = resolve`
-    div {
-        display: flex;
-        flex-direction: column;
-        max-height: calc(100vh - ${2 * spacersNum.dp64}px);
-        max-width: calc(100vw - ${2 * spacersNum.dp64}px);
-    }
-`
-
-const centeredContent = resolve`
-    .top {
-        padding-top: ${spacers.dp64};
-    }
-    .bottom {
-        padding-bottom: ${spacers.dp64};
-    }
-`
-
-const layerStyles = resolve`
-    div {
-		display: none;
-    }
-`
+const resolveLayerStyles = (hide) =>
+    resolve`
+        div {
+            padding: ${spacers.dp64};
+            display: ${hide ? 'none' : ''};
+        }
+    `
 
 export const Modal = ({
     children,
     className,
     dataTest,
     hide,
+    fluid,
     large,
     onClose,
     position,
     small,
 }) => {
+    const layerStyles = resolveLayerStyles(hide)
     return (
         <Layer
             onClick={onClose}
-            className={hide ? layerStyles.className : ''}
+            className={layerStyles.className}
```

Comment:
Although VERY unlikely, this could be a breaking change; therefore technically it is a breaking change

Reply:
The API has not changed, and all the changes are contained within the Modal components. I don't see how this could be a breaking change. Would you mind expanding on this ?

Reply:
The `hide` functionality is still handled in the `resolveLayerStyles` function: `display: ${hide ? 'none' : 'block'};` on L14.

Reply:
nvm.. My bad